scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -325,7 +337,22 @@ def create_lvm(self, index, metadata_dev):
     cmd = f"lvcreate -n {lv_md_name} {vg_name} -l 100%FREE --yes"
     execute_command(self, cmd)
 
+    swap_lvm_size_cmd = f"lvs /dev/{vg_name}/{lv_swap_name} -o LV_SIZE --noheadings --units M --nosuffix"
+    swap_lvm_size_op = execute_command(self, swap_lvm_size_cmd)
+    swap_lvm_size = int(float(swap_lvm_size_op[0].strip(' \n')))
+    swap_check_cmd = "free -m | grep Swap | awk '{print $2}'"
+    free_swap_op = execute_command(self, swap_check_cmd)
+    free_swap_size = int(float(free_swap_op[0].strip(' \n')))
+    free_swap_size = free_swap_size + swap_lvm_size
+    execute_command(self, "sleep 30")
     create_swap(self, swap_dev)
+    execute_command(self, "sleep 30")
+    allocated_swap_op = execute_command(self, swap_check_cmd)
+    allocated_swap_size = int(float(allocated_swap_op[0].strip(' \n')))
+    if free_swap_size >= allocated_swap_size:
```

Comment:
it should be otherway, isn't it,
allocated_swap_size > free_swap_size
i mean swap space after > swap space before

Reply:
Ideally (allocated_swap_size  <= free_swap_size) because free_swap_size is the expected size and allocated_swap_size is actual swap size. allocated_swap_size can be less due to not able to allocate enough swap space.

Reply:
free_swap_size is the total swap size we get from free command.
allocated_swap_size is the swap sizes we allocated in metadata disks.

Reply:
Corrected...de5164e